app/src/utilities.lisp

```diff
@@ -102,9 +102,19 @@
               (tbnl:session-remote-addr tbnl:*session*)
               (tbnl:session-id tbnl:*session*))))
 
-(defun format-log (level fmt-string &rest args)
-  (apply #'cl-syslog:format-log
-         *logger*
-         level
-         (concatenate 'string "~A" fmt-string)
-         (session-info) args))
+(defmacro format-log (level-or-fmt-string &rest fmt-string-or-args)
```

Comment:
this macro isn't quite right. We want the string and args to be evaluated. No need for `apply` either; we can splice in everything so it can be a normal function call

Reply:
they *are* evaluated

Reply:
``` 
(format-log :info "test ~A~A" 1 2)
<134>1 2019-04-01T21:55:21Z skilbeck-macbook.local qvm 30627 - - test 12
```

Reply:
oic what you mean

Reply:
as in `(format-log :info (concatenate 'string "~A" "~A") 1 2)`